client/setup.py

```diff
@@ -32,10 +32,10 @@ def run_tests(self):
         'Topic :: System :: Software Distribution',
         'Topic :: Utilities'
       ],
-      scripts=['main/dpcs'],
```

Comment:
No need to have two separate install files, settings should be installed always when dpcs is installed.